superset/db_engine_specs/hive.py

```diff
@@ -106,6 +106,31 @@ def fetch_data(cls, cursor: Any, limit: int) -> List[Tuple[Any, ...]]:
         except pyhive.exc.ProgrammingError:
             return []
 
+    @classmethod
+    def get_create_table_stmt(
+        cls,
+        table: Table,
+        schema_definition: str,
+        header_line_count: int,
+        null_values: Optional[List[str]],
+    ) -> text:
+        tblproperties = []
+        # available options:
+        # https://cwiki.apache.org/confluence/display/Hive/LanguageManual+DDL
+        if header_line_count > 0:
+            tblproperties.append(f"'skip.header.line.count'='{header_line_count}'")
+        if null_values:
+            # hive only supports 1 value for the null format
+            tblproperties.append(f"'serialization.null.format'='{null_values[0]}'")
+        tblproperties_stmt = ""
+        if tblproperties:
+            tblproperties_stmt = f"tblproperties ({', '.join(tblproperties)})"
```

Comment:
Can we write this in some way that lets us use `:params` for the table properties?

Reply:
issue is that tblproperties is optional - and it makes it quite tricky to implement, I did not figure it out. Open to ideas here

Reply:
Maybe we could make this function return `Tuple[text, Dict[str, str]]`. The code in the outer function would be something like
```python
sql, params = cls.get_create_table_stmt(...)
...
engine.execute(sql, **params)
```

Reply:
@serenajiang  done